core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,65 @@ var AdminPage = function(){
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="statusMessage"]'));
 
+  // The reload functions are used for mobile testing
+  // done via Browserstack. These functions may cause
+  // a problem when used to run tests directly on Travis.
+  if (general.isInDevMode()) {
+    var explorationElements = element.all(by.css(
+      '.protractor-test-reload-exploration-row'
+    ));
+
+    var reloadAllExplorationsButtons = element.all(by.css(
+      '.protractor-test-reload-all-explorations-button'
+    ));
+
+    var reloadCollectionButtons = element.all(by.css(
+      '.protractor-test-reload-collection-button'));
+
+    var explorationTitleElement = function(explorationElement) {
```

Comment:
Start functions with `get`. Ditto for explorationElementReloadButton.

Reply:
Done, thanks!